### DISCUSSION TITLE
Mod options?

### DISCUSSION THREAD

**Discussion by DOOMOlivera** [Tobi-Mob/Skul.Mod#22] [Q&A]
Before I install the mod i just want to know one thing. Can I install only the "hold the button to attack"? I don't really want to mod the game, it's just that I constantly feel a tingling in my thumb. Even while I'm not playing anything, and it's because of the button spam.

**Comment by Tobi-Mob** [comment]
Hi,

Only the auto attack is active be default.
Everything else needs to be enabled by pressing F3,F4 or F5
